src/features/observe/ObserveScreen.ts

```
@@ -540,7 +540,14 @@ export class RealObserveScreen implements ObserveScreen {
           };
         }
 
-        result.screenIdentity = deriveIosScreenIdentity(result.viewHierarchy);
+        let sdkScreenIdentity: ScreenIdentity | undefined;
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)